lib/web_ui/lib/src/engine/mouse_cursor.dart

```diff
@@ -15,8 +15,8 @@ class MouseCursor {
   }
 
   /// The [MouseCursor] singleton.
-  static MouseCursor get instance => _instance;
-  static MouseCursor _instance;
+  static MouseCursor? get instance => _instance;
+  static MouseCursor? _instance;
```

Comment:
late?

Reply:
Probably in a follow-up PR. The initialization is explicit via `static void initialize`, which needs a null check to initialize, so it can't be `late` without some code refactor.